sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.models.ts

```diff
@@ -17,4 +20,10 @@ export interface ScholasticStanding {
  */
 export interface ScholasticStandingSummary {
   totalUnsuccessfulWeeks: number;
+  offeringIntensity: OfferingIntensity;
+}
+
+export interface ScholasticStandingSummaryDetails {
+  fullTimeUnsuccessfulCompletionWeeks: number;
+  partTimeUnsuccessfulCompletionWeeks: number;
```

Comment:
Minor and not blocker, these property names sound better and can be used in DTO also.